pillar/master/config.sls

```diff
@@ -190,18 +105,13 @@ salt_master:
     misc:
       worker_threads: 25
       {# this is to avoid timeouts waiting for edx asset compilation during AMI build (TMM 2019-04-01) #}
-      gather_job_timeout: 30
-      master_job_cache: pgjsonb
-      event_return: pgjsonb
-      returner.pgjsonb.host: postgres-saltmaster.service.consul
-      returner.pgjsonb.port: 5432
-      returner.pgjsonb.user: __vault__:cache:postgres-operations-saltmaster/creds/saltmaster>data>username
-      returner.pgjsonb.pass: __vault__:cache:postgres-operations-saltmaster/creds/saltmaster>data>password
-      returner.pgjsonb.db: saltmaster
+      gather_job_timeout: 60
     sdb:
       consul:
         driver: consul
         host: consul.service.operations.consul
+      osenv:
+        driver: env
```

Comment:
?

Reply:
This is for using environment variables as an SDB backend.